src/Router.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import Calculator from './components/Calculator';
+import Home from './components/Home';
+import ErrorPage from './components/ErrorPage';
+import Navbar from './components/Navbar';
+import Quote from './components/Quote';
+
+function RouterNav() {
+  return (
+    <>
+      <BrowserRouter className="app-container">
+        <Navbar />
+        <Routes>
+          <Route exact path="home" element={<Home />} />
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job working on this milestone! 🚀 However, I would suggest that instead of defining this route's path as `home`, you change it to `/`. This is a common convention for the homepage of a site. At the moment, when you first enter your page (the path is `/`) you get an Error Page, and I'm pretty sure that's not what you want! 🤓 🔧

Reply:
Thanks for your advice @anagudelogu, i followed your suggestion. 💪🏽 !!